std_msgs/Header header
float64 targetX              # targetX reading
float64 targetX_variance     # targetX_variance reading

